common/network-common/src/main/java/org/apache/spark/network/shuffledb/DBBackend.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.network.shuffledb;
+
+import java.util.Locale;
+
+/**
+ * The enum `DBBackend` use to specify a disk-based store used in shuffle service local db.
+ * Only LEVELDB is supported now.
+ */
+public enum DBBackend {
+    LEVELDB(".ldb");
```

Comment:
is the intent to make it fully pluggable or only support the versions Spark directly provides?

Reply:
Only the version provided by Spark is supported, and will add `RocksDB` in a separate pr